content/articles/getting_started/1_setting_up_your_development_environment_unix.md

```diff
@@ -12,6 +12,16 @@ The only development environment that MonoGame officially supports on Linux is [
 ## Install .NET 6 SDK
 
 - .NET SDK at: [https://dotnet.microsoft.com/en-us/download](https://dotnet.microsoft.com/en-us/download)
+- If you are intending to target mobile platforms, make sure that you have the corresponding workload installed
+
+```sh
+dotnet workload install ios
+dotnet workload install android
+```
+
+## macOS considerations
+
+For the time being, MonoGame requires that you install the x64 version of the .NET SDK even if you are running on an Apple Silicon mac. It is also required that [Rosetta](https://support.apple.com/en-us/HT211861) is enabled.
```

Comment:
Even with Rosetta emulation, the mgcb will fail to build Textures and SpriteFonts due to the freeimage and freetype libs needing to be built for the M1/M2 architecture.  

There are workarounds that some have used such as doing a brew install of both libs and creating a symnlink for them